lib/components/table.vue

```diff
@@ -318,6 +320,29 @@
         opacity: 0.3;
     }
 
+    table.table-sm thead .sorting:before,
```

Comment:
Yeah, I was thinking the same.

Maybe making asc/desc their own class, and maybe making disabled its own class.